### ISSUE TITLE
Using this as a background service

### ISSUE THREAD

**Issue by drakula84** [wf9a5m75/plugin-socket.io#1]
Hi 

Great plugin.. thanks for that. I need to have permanent connection with the server.. is it possible to have this plugin run as a background service so that the service still runs if the app is killed? Or is there a better solution for that scenario?

**Comment by wf9a5m75** [comment]
Although this plugin does not run in service (because of I didn't design for it), the source code is able to run in service.
Create a service class, then copy the source code.

**Comment by drakula84** [comment]
Thanks, yes, I did it and it seems to work. However, https didn't work out of box. I will need to investigate it a bit further. I have seen this thread: https://github.com/nkzawa/socket.io-client.java/issues/26 - but I will read it tomorrow.

Had you tested it with https?

**Comment by wf9a5m75** [comment]
Yes, this library works fine with https. I uses https://github.com/nkzawa/socket.io-client.java in my work project.
Do you use self-signed ssl file, perhaps?

**Comment by drakula84** [comment]
Hi yes, I am using self-signed certificates, but I tried to install it on my device and it said certificate is installed. However, it doesnt list it under Security-Credential Storage-Trusted Credentials. I am not too sure if it is supposed to list it there anyway.

Should it work with self signed certs once they are installed?

**Comment by drakula84** [comment]
Using the SSL configuration code from page below makes it work: 
https://github.com/jonathanve/socket-io-android/blob/master/client-snippet.java

**Comment by servco50** [comment]
asda
